packages/atxp-base/README.md

```diff
@@ -0,0 +1,71 @@
+# ATXP Base
+
+ATXP is a framework for building and running agents that can interact with the world. See [docs.atxp.ai](https://docs.atxp.ai) for documentation and examples.
+
+ATXP Base provides a `BaseAppAccount` implementation for using `@atxp/client` with [Base Mini Apps](https://www.base.org/build/mini-apps).
+
+## Initializing the account
+In your Base Mini App's React code:
+```
+import { BaseAppAccount } from "@atxp/base";
+
+// This example uses wagmi to fetch the user wallet address, but any 
+// equivalent method that returns the address should work
+import { useAccount } from "wagmi"; 
+const { address } = useAccount();
+
+// Base mini app API key from the Coinbase Developer Portal
+const apiKey = process.env.NEXT_PUBLIC_ONCHAINKIT_API_KEY!;
+
+const account = await BaseAppAccount.initialize({
+  walletAddress: address,
+  apiKey,
+  appName: 'Mini App ATXP',
+  // Parameters for the spend permission (see How it works below)
+  allowance: BigInt('10000000'), // 10 USDC
+  periodInDays: 30,
+});
+
+// Then, use the account as you would any other Account - see
+// docs.atxp.ai for full docs and examples
+const client = atxpClient({
+  account,
+  mcpServer: 'https://browse.mcp.atxp.ai/'
+});
+const res = await client.callTool({
+  name: 'atxp_browse', 
+  {query: 'How do I make a mini app?'}
+);
+```
+
+
+## Clearing state
+You can reset the BaseAppAccount state for a user by:
+```
+BaseAppAccount.clearAllStoredData(address);
+```
+
+This will wipe any existing spend permission and ephemeral wallet for the given address. It will be automatically re-created the next time you call `BaseApAccount.iniitalize` for that address.
```

Comment:
```suggestion
This will wipe any existing spend permission and ephemeral wallet for the given address. It will be automatically re-created the next time you call `BaseAppAccount.iniitalize` for that address.
```
Fixed typo